docs/RestoreDatabase.md

```diff
@@ -0,0 +1,210 @@
+# Restore database guide
+
+This guide explains how to restore ScalarDB/ScalarDL data on a Kubernetes environment. We assume you use the managed databases provided by cloud services as a backend database of ScalarDB/ScalarDL.
+
+## Restore operations
+
+1. Scale in the Scalar product pods to **0** for stopping requests from Scalar products to backend databases.
```

Comment:
We can do the same thing by `helm uninstall` instead of scale-in the pods to 0. However, if we run the `helm uninstall`, the Load Balancer is also removed. In other words, from the perspective of client applications, the endpoint for Scalar products (Load Balancer's FQDN or IP for envoy pods) will be changed. In this case, users need to update the client application configurations to access the new Load Balancer.

Also, we can do the same thing by pausing Scalar pods using `scalar-admin`. This can avoid pod restart. However, basically, the endpoint of each managed database will be changed since the PIRT feature restores data as another database instance. So, we need to update `scalar.db.contact_points` in *.properties file. It requires restarting pods to apply this new `scalar.db.contact_points` configuration. In other words, we cannot avoid pod restart. So, I didn't select this way (pausing Scalar pods by `scalar-admin`).

So, I selected this way (scale-in Scalar pods to 0) to stop the requests from Scalar products to the backend databases when we restore databases. In this way, the Load Balancer is not removed. So, users don't need to update their client application configurations since the FQDN or IP for the envoy will not be changed.